spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -38,6 +38,8 @@
 import org.springframework.cloud.common.security.core.support.OAuth2AccessTokenProvidingClientHttpRequestInterceptor;
 import org.springframework.cloud.common.security.core.support.OAuth2TokenUtilsService;
 import org.springframework.cloud.common.security.support.SecurityStateBean;
+import org.springframework.cloud.dataflow.aggregate.task.AggregateExecutionSupport;
```

Comment:
These imports are still out of order. I would expect to see only green lines for the newly introduced contributor clasess.

Reply:
Lesson learned!! NEVER do `Optimize Imports` from IntelliJ, it messes up everything

Reply:
Lol. Yeh, unless you have the code styles setup you do not want to do this.